### DISCUSSION TITLE
[x][closed] track:global_id

### DISCUSSION THREAD

**Discussion by dzerbino** [fairtracks/fairtracks_standard#97] [Team Posts]
No infrastructure at the moment.

**Comment by sveinugu** [comment]
Nope. Should not be required, at least not until we possibly have implemented some sort of autogeneration of track ids.

**Comment by sveinugu** [comment]
Btw: We could relatively easily build a prototype of a track_id generation system if we wanted (very simple REST API), but not right now.

**Comment by dzerbino** [comment]
OK, so I'm dropping those.
